lib/colors.dart

```diff
@@ -92,3 +92,24 @@ const Color greyCardBackground = Color(0xfff7f7f9);
 const Color appBackgroundColor = Color(0xffF4F5FA);
 
 const Color containerBackgroundColor = Color(0xffF6F7FB);
+
+// buy/sell cover workflow colors
```

Comment:
Remove this comment because the colors won't be specific to only the marketplace.

Reply:
Noted